modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3476,6 +3476,71 @@ definition {
 			actionLabel = "Update Action Label");
 	}
 
+	@description = "On Commerce Product, add / view update action on System Objects using System Objects"
+	@priority = 5
+	test CanUpdateCommerceProductEntryAfterCreatingCommerceProductEntry {
+		property portal.acceptance = "true";
+
+		task ("Given Commerce Product object and given an action to update a Commerce Product entry after creating a Commerce Product entry") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Product");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				active = "true",
+				fieldName = "Name",
+				inputMethod = "true",
+				inputValue = "Shoes",
+				thenAction = "Update an Object Entry",
+				whenAction = "On After Add");
+		}
+
+		task ("When the user adds a new Commerce Product entry") {
+			CommerceProducts.openProductsAdmin();
+
+			Click(locator1 = "Button#PLUS");
+
+			Click(
+				key_menuItem = "Simple",
+				locator1 = "MenuItem#DROPDOWN_MENU_ITEM");
```

Comment:
You could replace this step with: `DropdownMenuItem.click(menuItem = "Simple");`